docs/source/reference/deprecation.rst

```diff
@@ -539,3 +539,56 @@ Schedule
   raise deprecation warnings.
 - In Numba 0.60: ``NUMBA_CAPTURED_ERRORS=new_style`` became the default.
 - In Numba 0.61: support for ``NUMBA_CAPTURED_ERRORS=old_style`` was removed.
+
+.. _cuda-target-deprecation-notice:
+
+Deprecation of the built-in CUDA target
+=======================================
+
+The CUDA target is now maintained in a separate package, `numba-cuda
+<https://nvidia.github.io/numba-cuda>`_, and the built-in CUDA target is
+deprecated.
+
+Reason for deprecation
+----------------------
+
+Development of the CUDA target as been moved to the ``numba-cuda`` package to
+proceed independently of Numba development.
```

Comment:
Maybe link back to the `_cuda-deprecation-status` anchor from the overview in the user docs which expands on the reasons?